### ISSUE TITLE
When trying to contribute to the docs, instructions don't work on Windows

### ISSUE THREAD

**Issue by andrewjtech123** [readthedocs/readthedocs.org#10304]
## Expected Result

Was hoping to contribute to the docs by following the instructions here:
https://dev.readthedocs.io/en/latest/docs.html

## Actual Result

The steps don't work on Windows. Requires WSL or update the steps so that the commands will work when run from Windows cmd.

**Comment by humitos** [comment]
I don't think we can commit ourselves to keep the guide updated to work on Windows (none of us use Windows), but at least, I think we should add a note of the expected requirements similar to what we have in our installation guide: https://dev.readthedocs.io/en/latest/install.html#requirements
